src/chttpd/src/chttpd_misc.erl

```diff
@@ -108,39 +108,57 @@ maybe_add_csp_headers(Headers, _) ->
     Headers.
 
 handle_all_dbs_req(#httpd{method='GET'}=Req) ->
-    % TODO: Support args and options properly, transform
-    % this back into a fold call similar to the old
-    % version.
-    %% Args = couch_mrview_http:parse_params(Req, undefined),
+    #mrargs{
+        start_key = StartKey,
+        end_key = EndKey,
+        direction = Dir,
+        limit = Limit,
+        skip = Skip
+    } = couch_mrview_http:parse_params(Req, undefined),
```

Comment:
I've been purposefully delaying moving code between modules if its not being changed. Eventually there's going to be a fairly massive purge of unused code and I think its probably best to wait until then that we start doing organizational changes rather than mix it in with new development.